src/resources/help/en_US/relnotes3.3.2.html

```diff
@@ -29,6 +29,7 @@
 <p><b><font size="4">Enhancements in version 3.3.2</font></b></p>
 <ul>
   <li>In the X Ray Image Query, a warning was added for the BMP output type, saying it may not function properly. This output type will be removed in the next major release of VisIt.</li>
+  <li>Added support for wedges and pyramids in the Blueprint plugin, for when VisIt uses a newer Conduit.</li>
```

Comment:
Is the plan to update conduit prior to 3.3.2 release? I couldn't find a ticket about it. If not, I think its confusing to have this note here because the feature isn't actually enabled due t o old Conduit.

Reply:
I don't know the answer to that. Should I remove the entry from the release notes then?

Reply:
I am inclined to recommend that, yes. Reason being is that I think its potentially confusing to announce an enhancement that cannot actually be realized by a user. Right time I think to put a note about it in release notes is when the release will support it for actual users.

I suspect we may not wanna update Conduit in a patch release so might have to put that off to 3.4. In that case, maybe add an issue ticket for that Conduit update, set it for 3.4 and then add this note to `relnotes3.4.0.html`

Reply:
What version of Conduit does it need? We are using 0.8.4 right now.

Reply:
@brugger1 It will be whatever version comes next (0.8.5 then?). I just added the support to conduit yesterday.

Reply:
I think the last time we upgraded to a new conduit was in 3.3.1. Maybe the better call would be to add a ticket to upgrade to the latest and greatest conduit, whenever it comes out, and then leave a note there to add to the release notes in whatever version we upgrade to newer conduit that we have added pyramid and wedge support. Does that sound reasonable?

Reply:
sounds good